backup.sh

```diff
@@ -159,6 +159,13 @@ function finish {
 trap finish EXIT
 
 
+# To help avoid a deluge of updates from every manta service immediately at the
+# top of every hour we first sleep for a random time between zero and thirty
+# minutes and then proceed with uploading the log files.
+sleepsecs=$((RANDOM % 1800))
```

Comment:
I'm assuming bash's RANDOM is Good Enough (TM).

Reply:
Yeah, in this case it's good enough. It doesn't need to be cryptographically strong, just fairly well distributed.